### PR TITLE
Create name class

### LINKED ISSUE #6: samandar ism yozjx

### PR DESCRIPTION
**Pull request by SamandarYigitaliyev**
closes #6

### CODE REVIEW

File: pullrecuest/SamandarbekYigitaliyev.cs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
```
Comment (rustambekovich) [inline pullrecuest/SamandarbekYigitaliyev.cs:5]: ha bu nimaee

File: pullrecuest/SamandarbekYigitaliyev.cs [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+namespace pullrecuest
+{
+    public class SamandarbekYigitaliyev
```
Comment (rustambekovich) [inline pullrecuest/SamandarbekYigitaliyev.cs:9]: ichi yoo buni

File: pullrecuest/SamandarbekYigitaliyev.cs (hunk 1 above)
Comment (SamandarYigitaliyev) [reply pullrecuest/SamandarbekYigitaliyev.cs:5]: nmaga oxshayapti

File: pullrecuest/SamandarbekYigitaliyev.cs (hunk 2 above)
Comment (SamandarYigitaliyev) [reply pullrecuest/SamandarbekYigitaliyev.cs:9]: oziz yozn

File: pullrecuest/SamandarbekYigitaliyev.cs [hunk 3]
```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace pullrecuest
+{
```
Comment (SamandarYigitaliyev) [inline pullrecuest/SamandarbekYigitaliyev.cs:8]: toldi

**Review by SamandarYigitaliyev — COMMENTED** [review]
nmaa gapp

File: pullrecuest/SamandarbekYigitaliyev.cs [hunk 4]
```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
```
Comment (rustambekovich) [inline pullrecuest/SamandarbekYigitaliyev.cs:3]: ha hammasi yaxsi

**Review by rustambekovich — APPROVED** [review]

**Pull request merged**